### ISSUE TITLE
measure fieldNames contains undesirable values

### ISSUE THREAD

**Issue by merlijnbuit** [tableau/extensions-api#241]
Hi,

Don't know if this is already mentioned here but it would be really helpful to retrieve the column name without any aggregation around. We have to remove the AGG( ) or SUM( ) from the fieldNames currently with some regex. But it would be nice to just have the name of the column to reduce code complexity and possible mistakes with different languages.

Current:
fieldName: AGG(Profit Ratio)

Desirable:
fieldName: Profit Ratio
aggType: AGG

![image](https://user-images.githubusercontent.com/16118919/60101941-f97cc580-975c-11e9-9d41-ff0c8a6b6e09.png)

**Comment by Kovner** [comment]
Yep this makes sense. We are going to look into adding the ability to reference fields, filters, etc. by id. I think this would fit into that rethinking of how we refer to objects.

**Comment by johnDance** [comment]
@merlijnbuit - Has our work adding fieldId to column solved this issue, or are there additional holes?
John
